k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml

```diff
@@ -144,10 +145,12 @@
 - name: nginx-errors
   dmap:
     "sha256:5530c7fc1fa75b49e068d6b7b4f8f1c13a5834c155ba7abd5943b9057c3b925b": ["0.48.1"]
+    "sha256:06cd2dbe317505e940d0521cc69d237752800ef0b191cb4265eb0d3d2c1080e7": ["v20220415-controller-v1.2.0-beta.0-2-g81c2afd97"]
```

Comment:
0.49.0 here :)

Reply:
Changed to 0.49.0 on line 148 for Nginx-errors image
I was following the cloudbuild info here
<img width="908" alt="image" src="https://user-images.githubusercontent.com/5085914/163658169-079bf11b-addc-4a00-acf2-a38beb4b1131.png">